src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -62,9 +62,14 @@ trait TestDriverComponent extends DriverComponent with DataAccess {
       case ref: Entity => Workflow(UUID.randomUUID.toString, WorkflowStatuses.Submitted, testDate, Option(AttributeEntityReference(ref.entityType, ref.name)), inputResolutions(ref))
     })
 
+    val failedWorkflows = (workflowEntities collect {
```

Comment:
yeah, it doesn't make a lot of sense for each entity to have both a success and a failure. I'll see if I can refactor that.

It doesn't trigger the exception because this (awkward test case) has a single entity with both a success resolution and a failure resolution; the exception is triggered if a single resolution has both a successful workflow and a failed workflow.